etc/helm/pachyderm/values.yaml

```diff
@@ -239,6 +239,51 @@ loki-stack:
     config:
       lokiAddress: http://{{ .Release.Name }}-loki:3100/loki/api/v1/push
       snippets:
+        scrapeConfigs: |
```

Comment:
could we add a comment here to describe the backstory and what we added vs what we copied?

Reply:
Sure thing